vault/login_mfa.go

```diff
@@ -1601,6 +1602,11 @@ func parseOktaConfig(mConfig *mfa.Config, d *framework.FieldData) error {
 		oktaConfig.PrimaryEmail = true
 	}
 
+	usePasscode := d.Get("use_passcode").(bool)
+	if usePasscode {
+		oktaConfig.UsePasscode = true
+	}
```

Comment:
Similar to how the duo MFA parses config:
https://github.com/hashicorp/vault/blob/58a5b77bc6f0a5ac47e3e3d6020148b585f7213a/vault/login_mfa.go#L1202